lib/foods.js

```diff
@@ -4,37 +4,92 @@
 const herokuUrl = require("./heroku-url")
 const $ = require("jQuery")
 
+$('#newFoodForm').hide();
+
 function getFoods() {
   //counter = 1;
   $.ajax({
     type: "GET",
     url: herokuUrl() + "api/v1/foods",
     success: function(posts) {
-      posts.forEach(function(foodType){
+      posts.reverse().forEach(function(foodType){
         event.preventDefault;
         let button = `<td><button type='button' class='deleteButton' id='${foodType.id}'>Delete</button></td>`
           let toInsert = `<tr id=${foodType.id}><td>${foodType.name}</td><td>${foodType.calories}</td>${button}</tr>`;
-          $(".foodsTable").append(toInsert);
+        $(".foodsTable").append(toInsert);
```

Comment:
@bretfunk I use the same posting to a table in the foods filter function that we can probably reuse. I think lines 15-19 can be refactored. Let's Talk-o 🌮